package.json

```diff
@@ -3,11 +3,15 @@
   "description": "Project template for Drupal 8 themes",
   "version": "1.0.0",
   "devDependencies": {
-    "emulsify-gulp": "fourkitchens/emulsify-gulp",
+    "emulsify-gulp": "fourkitchens/emulsify-gulp#stylelint",
```

Comment:
This is temporary for reviewing the work. Once it works we'll merge the Emulsify Gulp PR and update this package.json.